### ISSUE TITLE
"CHAT" SPAM, STUPID QUESTIONS ABOUT PROXY ETC...

### ISSUE THREAD

**Issue by l3mpik** [l3mpik/slither-feeder-bot#17]
blah, blah,,,

**Comment by kurama129** [comment]
answer😀

**Comment by l3mpik** [comment]
lol :dancer:

**Comment by kurama129** [comment]
lol what

**Comment by l3mpik** [comment]
idk

**Comment by kurama129** [comment]
i have a im creating best slither bots than this rewriting everything :)

**Comment by kurama129** [comment]
ill post when done i got quick respawn and eat food and speed up  features

**Comment by anonymousxxx02** [comment]
@kurama129 can  i test that script pleaseee...i wont share it i just want to test it

**Comment by l3mpik** [comment]
Eat food options good idea . Wanna work together on this ? If no easy i understeand : P

**Comment by easyrider81** [comment]
why is the bot counter so unreliable? may aswell not have it..

**Comment by anonymousxxx02** [comment]
@l3mpik did you add random name  in the code???

**Comment by CupidStunts** [comment]
some spam lol
https://imgur.com/5a8J703
then died by my own bots :(

we do need a better bot counter! i had so many then it was like half the server was bots lol

**Comment by l3mpik** [comment]
I waiting for client update than i make this better. @anonymousxxx02  Yes, random name with random skin.

**Comment by MineMooreALT** [comment]
How do fix max bots 70 i wanna make max bot 1000

**Comment by l3mpik** [comment]
haha, get good proxy : )

**Comment by l3mpik** [comment]
Download proxies.txt from this repo . 80-100 bots. I upload now.

**Comment by MineMooreALT** [comment]
I did that goses up to 100  for split second drops to 5 after

**Comment by MineMooreALT** [comment]
where do you get ur proxys

**Comment by l3mpik** [comment]
Http Proxy blog spot <--- Use google. Yee, i know i drop. But when u agin click on bots. U get agin <100 . I wait for better client

**Comment by MineMooreALT** [comment]
i'll try thanks

**Comment by MineMooreALT** [comment]
turns out it was the same one i used before

**Comment by l3mpik** [comment]
Ohh.. : )

**Comment by MineMooreALT** [comment]
But they dont work well for me can you send me the specfic website maybe just cant see the one you are talking about

**Comment by l3mpik** [comment]
http://proxyserverlist-24.blogspot.com/ 

+

http://www.checker.freeproxy.ru/checker/

Good Luck! : )

**Comment by ghost** [comment]
client and server  done adding features to it and tell me if i should post and let youtubers steal my work or stay private and btw ill add a bot counter as you said it shows how many bots are in the server

**Comment by ghost** [comment]
dahquan collide is releasing

**Comment by ghost** [comment]
added uuid as many requested

**Comment by ghost** [comment]
added reconnect

**Comment by ghost** [comment]
but still less bots

**Comment by ghost** [comment]
anyways im getting new lib

**Comment by wartoz** [comment]
screenshot?

**Comment by wartoz** [comment]
http://image.noelshack.com/fichiers/2016/21/1464441269-skypeitwigxdevbots.png

**Comment by ghost** [comment]
wartoz why you selling free ones for money

**Comment by wartoz** [comment]
https://giphy.com/gifs/3o6EhWnYy8ZIhlYhc4 bots gif in-game

**Comment by ghost** [comment]
wartoz why you selling free ones for money

**Comment by wartoz** [comment]
selling ??

**Comment by wartoz** [comment]
it's leaked by slitherfree tk there is nothing to sell

**Comment by ghost** [comment]
oh ok

**Comment by ghost** [comment]
are you using free bots

**Comment by wartoz** [comment]
i'm using rattlesnake lib

**Comment by ghost** [comment]
yea but did you code client and servet

**Comment by ghost** [comment]
and that gui even i have

**Comment by anonymousxxx02** [comment]
@l3mpik bro slither.io update skin so you have to change skin number from 39 to 41 also remove the GUI it was better without it

**Comment by MineMooreALT** [comment]
update proxy please

**Comment by anonymousxxx02** [comment]
@MineMooreALT update your proxy every day from here http://www.socks24.org/

**Comment by anonymousxxx02** [comment]
@l3mpik please remouve this GUI i cant even start bots with random name and skin by this GUI

**Comment by easyrider81** [comment]
Yup remove GUI please. Can't start bots with yours so I use my own.

**Comment by l3mpik** [comment]
Today update sorry for problems.

**Comment by anonymousxxx02** [comment]
@easyrider81 if i give you RW permision in my c9.io can you upload meyed bots there???

**Comment by easyrider81** [comment]
That's too much trouble, i've uploaded it to mediafire so just grab it and whoever else wants it.

[https://www.mediafire.com/?2blb5bw7dfb4xe1](url)

**Comment by anonymousxxx02** [comment]
@easyrider81 the file doesnt exist :0

**Comment by anonymousxxx02** [comment]
try this site to send me the files http://www.dropsend.com/
send to anonymousx023x@gmail.com

**Comment by easyrider81** [comment]
link works fine for me sigh. 

[http://myaccount.dropsend.com/file/38a96f7309771413](url)

**Comment by anonymousxxx02** [comment]
its say error 404 file doesn exist

**Comment by easyrider81** [comment]
copy paste the link instead of click and it works

**Comment by anonymousxxx02** [comment]
@easyrider81 thanks bro now please  remove link before someone find and share this

**Comment by easyrider81** [comment]
Ok no problem. I've removed the links but if anyone else requests from me the files i'll gladly post again.

**Comment by anonymousxxx02** [comment]
@easyrider81 ok

**Comment by anonymousxxx02** [comment]
@l3mpik are you gonna make any update or no???

**Comment by l3mpik** [comment]
Yes :dancer:
